frontend/src/metabase/routes.jsx

```diff
@@ -164,6 +164,18 @@ export const getRoutes = store => (
       }}
     />
 
+    {/* AUTH */}
+    <Route path="/auth">
+      <IndexRedirect to="/auth/login" />
+      <Route component={IsNotAuthenticated}>
+        <Route path="login" title={t`Login`} component={LoginApp} />
+        <Route path="login/:provider" title={t`Login`} component={LoginApp} />
```

Comment:
From testing, since this is moved out of the `{/* APP */}` route. `IsNotAuthenticated` will not work since we're not loading the user into redux store yet. If we go to `http://localhost:3000/auth/login` after logged in we will stay on the login page, but previously if we already logged in `/auth/login` will redirect us to `/` (home page) instead.

Reply:
Added a e2e test for this also